cloudformation/slack-router-lambda.js

```diff
@@ -0,0 +1,44 @@
+const AWS = require('aws-sdk')
+
+function parseBody(event) {
+  const eventJSON = JSON.stringify(event.body, null, 2)
+  const eventBody = Buffer.from(eventJSON, 'base64').toString('ascii')
+  const bodyArray = eventBody.split('&')
+
+  const bodyObject = bodyArray.reduce((accumulator, currentValue) => {
+    const [key, value] = currentValue.split('=')
+
+    accumulator[key] = value
+
+    return accumulator
+  }, {})
+
+  return bodyObject
+}
+
+exports.handler = async (event) => {
+  const eventBody = parseBody(event)
+  const command = decodeURIComponent(eventBody.command).substring(1)
+
+  const params = {
+    Message: JSON.stringify(eventBody),
+    Subject: 'SNS from Slack Slash Command',
+    TopicArn: `arn:aws:sns:us-east-1:670261699094:${command}`,
```

Comment:
Is there a way to pass the account ID and region via environment parameters and remove the hardcoded string here?

Reply:
is it cf.accountId and cf.region? (https://github.com/mapbox/cloudfriend#pseudo-parameters)

Reply:
Done here: https://github.com/hotosm/slack-bots/commit/d6ff1455607912f566f606f5d6148cb9ec818d68

Seems like the AWS Region is actually already an environment variable for Lambdas :)